framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/scenario/WorkbenchWithKieServerScenarioImpl.java

```diff
@@ -87,6 +88,13 @@ public WorkbenchWithKieServerScenarioImpl(OpenShiftController openShiftControlle
         } catch (MalformedURLException e) {
             throw new RuntimeException("Malformed URL for workbench", e);
         }
+        String secureRouteHostWorkbench = project.getService(workbenchDeployment.getSecureServiceName()).getRoute().getRouteHost();
+        String secureUrlWorkbench = "https://" + secureRouteHostWorkbench + "/kie-wb.jsp";
```

Comment:
Does it work for REST tests? I am quite surprised that this "/kie-wb.jsp" works with REST.